compiler/src/pretyping.ml

```diff
@@ -252,6 +273,13 @@ module Env : sig
     val clear_locals : 'asm env -> 'asm env
   end
 
+  module TypeAlias : sig 
+
+    val push : 'asm env -> A.pident -> P.pty -> 'asm env
+    val get : 'asm env -> A.pident -> (P.pty L.located)
+
+    val get_opt : 'asm env -> A.pident -> (P.pty L.located) option
```

Comment:
This function (`get_opt`) is never called.

Reply:
corrected